script/buildSQLGeom.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+
+"""
+This module will accept an input text file with (x,y,z) coordinates in a CSV-format list
```

Comment:
I've got a working path to the USGS API implemented, but now I've come across the google elevation API which has global support. 
https://developers.google.com/maps/documentation/elevation/intro

We'd need a [google maps API key](https://developers.google.com/maps/documentation/elevation/get-api-key) and are subject to traffic quota constraints, but this supports passing a set of points (e.g. a path or line) and returning sampled elevations from along that path. It'll even generate altitude histograms for you if we wanted to get cute.

My guess is that since this is for open research the API key should be no problem and since we're not building some kind of high-frequency mobile app, I doubt traffic volume would be a concern.